### ISSUE TITLE
chezmoi diff runs very slow pass command

### ISSUE THREAD

**Issue by raffomania** [twpayne/chezmoi#852]
## Describe the bug

When I run `chezmoi diff` or `chezmoi apply` on my dotfiles, which includes only one call to the `pass` template helper, chezmoi runs the command `pass grep ^$` - which is very slow to finish (about 30 seconds). I've taken a short look at the source code and see no apparent reason for this to actually happen... no idea where it comes from!

## To reproduce

Include the following in a .tmpl file:
```
password = "{{ pass "foo@example.com" }}"
```

When this file is not present, everything is blazing fast!

## Expected behavior

`chezmoi diff` should take way less time to complete.

## Output of command with `--verbose`

```
$ chezmoi command --verbose
2020/08/17 20:23:29 IdempotentCmdOutput("/usr/bin/pass show 'foo@example.com'") (130.920145ms)
```

## Output of `chezmoi doctor`

```
$ chezmoi doctor
warning: version 1.8.4
     ok: runtime.GOOS linux, runtime.GOARCH amd64
     ok: /home/rafael/.local/share/chezmoi (source directory, perm 700)
     ok: /home/rafael (destination directory, perm 700)
     ok: /home/rafael/.config/chezmoi/chezmoi.toml (configuration file)
     ok: /bin/zsh (shell)
     ok: /usr/bin/vim (editor)
     ok: /usr/bin/vimdiff (merge command)
     ok: /usr/bin/git (source VCS command, version 2.28.0)
     ok: /usr/bin/gpg (GnuPG, version 2.2.21)
warning: op (1Password CLI, not found)
warning: bw (Bitwarden CLI, not found)
warning: gopass (gopass CLI, not found)
warning: keepassxc-cli (KeePassXC CLI, not found)
warning: lpass (LastPass CLI, not found)
     ok: /usr/bin/pass (pass CLI, version 1.7.3)
warning: vault (Vault CLI, not found)
```

**Comment by twpayne** [comment]
tl;dr this should be fixed with #850. It's merged into `master` but not yet released. Are you able to build chezmoi from source and test whether this fixes the problem for you?

Longer explanation:

Do you have a lot of data stored in pass?

chemoi runs `pass grep ^$` to force pass to unlock the keyring, but this has the side effect of decrypting everything. See #784 for the background on this.

#850 is a better fix that avoids the downsides.

cc @Grimler91

**Comment by raffomania** [comment]
awesome, built the master and it is super fast again :) thanks a lot!
